test/LocalizationSample.Mvc.FunctionalTest/HomeControllerSpec.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace LocalizationSample.Mvc.FunctionalTest
+{
+    public class HomeControllerSpec : IClassFixture<WebApplicationFactory<Startup>>
+    {
+        private readonly HttpClient _client;
+
+        public HomeControllerSpec(WebApplicationFactory<Startup> factory)
+        {
+            _client = factory.CreateClient();
+        }
+
+        [Fact]
+        public async Task PrivacyPolicy_should_return_Privacy_Policy()
+        {
+            var response = await _client.GetAsync("/Home/Privacy").ConfigureAwait(false);
+            var result = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+
+            result.Should().Contain("Privacy Policy");
+        }
+
+        [Fact]
+        public async Task PrivacyPolicy_should_return_Politique_de_confidentialité()
+        {
+            var response = await _client.GetAsync("/Home/Privacy/?culture-fr-FR&ui-culture=fr-FR")
```

Comment:
`?culture-fr-FR` -> `?culture=fr-FR`

Reply:
I surprised too, I saw that in the [image](https://github.com/hishamco/My.Extensions.Localization.Json/issues/43#issuecomment-608332510) you've sent me on the issue discussion. :+1: